### DISCUSSION TITLE
Bug in redirection after "I am human" dialog

### DISCUSSION THREAD

**Discussion by ovanderzee** [npm/feedback#377] [Website] [answered]
When you view a list of packages, e.g. the dependents of of a package and quickly open a number of these, you get a "I am human" dialog with a image puzzle. 
Now when the dependent is from an organisation and thus has a '@' symbol in the name, the url appears to be encoded:
e.g. https://www.npmjs.com/package/%40bolt%2Fjest-config.
When the image puzzle is solved, you are redirected to the 404 page, instead of the expected https://www.npmjs.com/package/@bolt/jest-config.

**Comment by MylesBorins** [comment, marked as answer]
@ovanderzee would you be able to provide a video / gif of this bug happening? I was not able to reproduce

**Reply by ovanderzee** [reply]
It appears to be fixed now. I filed support Ticket ID: 1137635 too
